ansible/kubernetes.yml

```diff
@@ -140,6 +140,10 @@
       tags:
         - initialize_kubernetes_cluster
         - quick_setup
+    - name: Enable kubectl autocompletion (master)
+      shell: |
```

Comment:
This is not idempotent. You should use the [lineinfile module](https://docs.ansible.com/ansible/latest/modules/lineinfile_module.html)